blogs/5_how_to_gorilla.html

```diff
@@ -0,0 +1,502 @@
+                                                }, 
+                                                {
+                                                    "name":"type", 
+                                                    "enum": ["plus", "comfort", "black"], 
+                                                    "description": "types of uber ride user is ordering"
+                                                },
+                                                {
+                                                    "name": "time", 
+                                                    "description": "the amount of time in minutes the customer is willing to wait"
+                                                }
+                                            ]
+                                        }
+                                </code></pre>
+                            </li> 
+                            <li> Define a response function (example below):
+                                <pre><code>
+                                    def get_gorilla_response(prompt="Call me an Uber ride type \"Plus\" in Berkeley at zipcode 94704 in 10 minutes", model="gorilla-openfunctions-v0", functions=[]):
+                                        openai.api_key = "EMPTY"
+                                        openai.api_base = "http://luigi.millennium.berkeley.edu:8000/v1"
+                                        try:
+                                            completion = openai.ChatCompletion.create(
+                                            model="gorilla-openfunctions-v1",
+                                            temperature=0.0,
+                                            messages=[{"role": "user", "content": prompt}],
+                                            functions=functions,
+                                            )
+                                            return completion.choices[0].message.content
+                                        except Exception as e:
+                                            print(e, model, prompt)
+                                </code></pre>
+                            </li>
+                            <li> Return relevant API call:
+                                <br><code>get_gorilla_response(query, functions=functions)</code>
+                            </li>
+                            <li> Output Example:
+                                <br><code>uber.ride(loc="berkeley", type="plus", time=10)</code>
+                            </li>
+                        </ol>
+                    </p>
+
+                    <p>
+                        Also, checkout the following tutorials for running and self hosting OpenFunctions locally!
+                        <ul>
+                            <li> <a href="https://colab.research.google.com/drive/1I9UJoKh9sngE2MfPfQD5kbn2-twq2xvY?usp=sharing">Run OpenFunctions Locally!</a>
```

Comment:
Maybe a line or 2 on what this is. I know it sounds mostly self explanatory but good to have for completion :)